fluentui_core/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Should we consider just one .gitignore file, instead of every module having its own?
Not sure of the recommended way. But to check gitignore at multiple locations will be cumbersome

Reply:
Done. I'll be removing .gitignore files in all subdirectories in next push